extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -214,6 +214,10 @@ public function adContextDataProvider() {
 				'expectedTargeting' => [ 'newWikiCategories' => [ 'test3' ] ],
 				'categories' => [ 'new' => [ 'test3' ] ]
 			] ),
+
+			array_merge( $defaultParameters, [
+				'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ], 'hasPoratableInfobox' => true ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "hasPoratableInfobox" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)